legal-api/src/legal_api/services/warnings/business/business_checks/corps.py

```diff
@@ -0,0 +1,47 @@
+# Copyright © 2022 Province of British Columbia
```

Comment:
2024

Reply:
Will do this on the next commit (Test cases)

Reply:
Done.